spec/features/work_packages/details/relations/primerized_relations_spec.rb

```diff
@@ -365,7 +365,7 @@ def label_for_relation_type(relation_type)
 
       wait_for_network_idle
 
-      expect(page).to have_text "Related work package not found or not visible"
+      expect(page).to have_text "Related work package not found."
```

Comment:
It feels like the specific error messages for `from_id` and `to_id` should be restored ([these lines](https://github.com/opf/openproject/commit/a12dbc69ba5a0b389d9e56a55a1f31fcc4818839#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R1521-L1526))

Reply:
As seen with Oliver, "We had the same string for a base not_found error in many other models, and we do not convey whether they are truly not found or not visible, hence leaning to that." so it makes sense to have the same generic message everywhere. The error messages should not be restored.